python/requirements.txt

```diff
@@ -11,3 +11,6 @@ aiohttp==2.2.5
 aiohttp_cors==0.5.3
 cchardet==2.1.0
 requests==2.18.4
+
+# sensortag
+pexpect==4.0.1
```

Comment:
This raises an interesting question about how we deal with requirements for plugins. Ultimately we do not what one requirements file that is a superset of all the plugins that may be used. However for now I think we just raise that as a Jira for future consideration.